backend/utils/sellers.js

```diff
@@ -14,20 +15,23 @@ async function createSeller({ name, email, password, superuser }) {
     return { status: 409, error: 'Registration exists' }
   }
 
-  const numSellers = await Seller.count()
   const salt = await createSalt()
   const passwordHash = await hashPassword(salt, password)
 
   const seller = await Seller.create({
     name,
     email,
```

Comment:
Should email get lowercased when storing it since line 11 above relies on that?

Reply:
Now lowercasing all emails, both for reading and for writing. Postel's law.